src/user/user.spec.js

```diff
@@ -37,7 +37,9 @@ describe("user", () => {
       async data => {
         const userId = await xo.createUser(data);
         expect(typeof userId).toBe("string");
-        expect(omit(await xo.getUser(userId), "id")).toMatchSnapshot();
+        expect(await xo.getUser(userId)).toMatchSnapshot({
+          id: expect.any(String),
+        });
```

Comment:
What about https://github.com/vatesfr/xo-test/pull/98/files#diff-5281a2d58525d7adc144f03d1fcee870R140 ?

Reply:
My bad, I forgot it.